src/mock/constants.ts

```diff
@@ -8,4 +8,5 @@ export const commands = {
     retrieveTestAppChannel: "retrieveTestAppChannel",
     broadcastInstrumentContext: "broadcastInstrumentContext",
     broadcastContactContext: "broadcastContactContext",
+    joinUserChannelOne: "joinUserChannelOne"   // v2.0
```

Comment:
`joinUserChannelOne` was recently changed to `joinRetrievedUserChannel` as the channel will vary depending on the desktop agent